source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -179,6 +179,15 @@ primary shard to each database by means of a round-robin.
    restart the migration from the start. For more information, see
    :ref:`c2c-sharded-limitations`. 
 
+Config Shard Cluster
+''''''''''''''''''''
+
+Starting in 8.0, MongoDB introduces support for :ref:`config-shard-concept`.
+
+``mongosync`` supports sync from dedicated config server sharded clusters to 
+config server sharded clusters and vice versa. Additionally, ``mongosync`` 
```

Comment:
sorry for the confusion, re-reading this now I think we do need "embedded" here to draw distinction with "dedicated" for clarity. 
We can say "sync from dedicated config server sharded clusters to embedded config server sharded clusters" here and add my above comment as well.